contrib/cmd/memfd-bind/README.md

```diff
@@ -1,6 +1,15 @@
 ## memfd-bind ##
 
-`runc` normally has to make a binary copy of itself when constructing a
+> **NOTE**: Since runc 1.2.0, runc will now use a private overlayfs mount to
+> protect the runc binary. This protection is far more light-weight than
+> memfd-bind, and for most users this should obviate the need for `memfd-bind`
+> entirely. Rootless containers will still make a memfd copy (unless you are
+> using `runc` itself inside a user namespace -- a-la
+> [`rootlesskit`][rootlesskit]), but `memfd-bind` is not particularly useful
+> for rootless container users anyway (see [Caveats](#Caveats) for more
+> details).
```

Comment:
Can you document the Linux kernel requirements?

Reply:
@rata are you referring to kernel requirements needed for overlayfs-based protection, or is it about memfd-bind?

Reply:
Oh, sorry, I mean overlayfs. We are using the new mount API, now another "new" syscall too. It's important to document it, IMHO.